packages/scripts/scripts/test-playwright.js

```diff
@@ -26,19 +26,25 @@ const {
 	getArgsFromCLI,
 } = require( '../utils' );
 
-const result = spawn(
-	'node',
-	[
-		path.resolve( require.resolve( 'playwright-core' ), '..', 'cli.js' ),
-		'install',
-	],
-	{
-		stdio: 'inherit',
-	}
-);
+if ( ! process.env.CI ) {
+	const result = spawn(
+		'node',
+		[
+			path.resolve(
+				require.resolve( 'playwright-core' ),
+				'..',
+				'cli.js'
+			),
```

Comment:
Off-topic: This is probably an implementation detail, I wonder why we don't just use `npx` here?

Reply:
I tried swapping it here but couldn't get it working locally - https://github.com/WordPress/gutenberg/pull/54051#discussion_r1387632055.

Reply:
Yeah I recall seeing this earlier but couldn't remember where 😆.

Reply:
It seems to be working for me via:

```js
const result = spawn( 'npx', [ 'playwright', 'install' ], {
	stdio: 'inherit',
} );
```